internal/services/synapse/synapse_sql_pool_extended_auditing_policy_resource_test.go

```diff
@@ -405,9 +408,9 @@ resource "azurerm_monitor_diagnostic_setting" "test" {
     }
   }
 
-  // log, metric will return all disabled categories
+  // enabled_log, metric will return all disabled categories
   lifecycle {
-    ignore_changes = [log, metric]
+    ignore_changes = [enabled_log, metric]
```

Comment:
do we need to `ignore_changes` on `enabled_log` here? I think switching from `log` -> `enabled_log` might remove the need for that

Reply:
Good shout! Taken care of. Ty ty